### PR TITLE
feedback

### LINKED ISSUE #73: As a user, I would like to be able to submit feedback for the admins

### PR DESCRIPTION
**Pull request by ghost**
fixes #73

### CODE REVIEW

File: API/api-handler.js [hunk 1]
```diff
@@ -247,6 +248,11 @@ exports.initialize = initialize;
 exports.isActiveSession = common_api.isActiveSession;
 // </Common Requests> -----------------------------------------------
 
+// <Feedbacks Requests> ------------------------------------------------
+exports.handleFeedbackCreatePath = feedback_api.createFeedback;
+exports.handleFeedbackAdminPath = feedback_api.renderFeedbackPage;
+// </Feedbacks Requests> -----------------------------------------------
```
Comment (SergeyGV) [inline API/api-handler.js]: alphasort

File: Backend/db.js [hunk 2]
```diff
@@ -82,6 +84,12 @@ exports.getComment = db_comments.getComment;
 exports.updateComment = db_comments.updateComment;
 // </Comments Collection> ---------------------------------------------
 
+// <Feedbacks Collection> ----------------------------------------------
+exports.addFeedback = db_feedback.addFeedback;
+exports.getLimitedFeedbacksListSorted = db_feedback.getLimitedFeedbacksListSorted;
+exports.getFeedback = db_feedback.getFeedback;
+// </Feedbacks Collection> ---------------------------------------------
```
Comment (SergeyGV) [inline Backend/db.js]: alphasort

File: Backend/dbs/db-feedback.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+        return callback(null, obj);
+    });
+}
+
+// <exports> -----------------------------------
+exports.addFeedback = addFeedback;
+exports.getLimitedFeedbacksListSorted = getLimitedFeedbacksListSorted;
+exports.getFeedback = getFeedback;
+exports.initialize = initialize;
```
Comment (SergeyGV) [inline Backend/dbs/db-feedback.js]: alphasort

**Review by SergeyGV — APPROVED** [review]
Nice work!

**Review by esteeroramy — APPROVED** [review]

**Pull request merged**